worldedit-core/src/main/java/com/sk89q/worldedit/regions/CuboidRegion.java

```diff
@@ -706,7 +726,7 @@ public IChunkSet processSet(IChunk chunk, IChunkGet get, IChunkSet set) {
 
         if (bx >= minX && tx <= maxX && bz >= minZ && tz <= maxZ) {
             // contains all X/Z
-            if (minY <= 0 && maxY >= 255) {
+            if (minY <= world.getMinY() && maxY >= world.getMaxY()) {
```

Comment:
we need a null-check on world

Reply:
Why would this be null? On versions prior to 1.17, it will return 0,255 still, just differentiating the outcome on worlds with custom heights. Tho, this does nothing atm as Fawe doesn't process these edits anyway.

Reply:
world in super class is nullable. (or... not not nullable)